### ISSUE TITLE
I'm completely newbie

### ISSUE THREAD

**Issue by siusiak** [askmike/gekko#1296]
As you can read in title i don't know any programming languages and  i would ask for forgiveness :)
I wrote some code, but i'm sure that is wrong and i would like to write the correct form. I think, that you understand it, but i don't know how to transform it.

https://pastebin.com/sE7tTxQs

**Comment by askmike** [comment]
could you post your code in a pastebin or a gist (gist.github.com)? Thanks!

**Comment by siusiak** [comment]
sure, edited :)

**Comment by askmike** [comment]
So your if checks are very wrong:

> `if('EMA1' > 'EMA2', 'FastK' = 20 && 'FastK' > 'SlowD')`

commas (`,`) do something else in javascript. I think the easiest would be to write out what you want to happen in plain english like:

    if EMA1 value is higher than EMA2 value AND
    if fastK is ....

And go to any online community where people help each other out when it comes to basis javascript scripting.

I would recommend posting your question to stackoverflow or something. Note that regardless of the ifcheck you probably want to move all your code that is now inside your `update` method into the `check` method.

**Comment by askmike** [comment]
Maybe someone wants to help you out in this issue tracker, but the first thing would be you explaining how you want your strategy to behave (`if this value is higher than that value OR bla bla`).

**Comment by siusiak** [comment]
It's known and simple strategy: there are three indicators: 2x EMA and stochastic oscillator, which consists of K and D values. The first EMA consists of 50 periods, and the second one consists of 100 periods. When value of first EMA is higher than second EMA and when K value is 20 and it;s higher than D value, long position is opened.

In oposition: first EMA< second EMA and K=80 and K value is lower than D, short position is opened.

I reached the point where there is no syntax error, but still i get:

 Error: non-error thrown: Child process has died.
      at Object.onerror (/home/shit/code/gekko/node_modules/koa/lib/context.js:105:40)
      at process._tickCallback (internal/process/next_tick.js:109:7)
